config/migrations/2023/20230925125154-remove-lpdc-data/20230925125154-delete-lpdc-your-europe-category-codelists.sparql

```diff
@@ -0,0 +1,150 @@
+PREFIX dvcs:  <https://productencatalogus.data.vlaanderen.be/id/conceptscheme/>
+PREFIX dvc:   <https://productencatalogus.data.vlaanderen.be/id/concept/YourEuropeCategorie/>
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?s ?p ?o .
+  }
+}
+WHERE {
+  VALUES ?s {
```

Comment:
Wouldn't it be easier to just remove the code list and the related concepts?

Reply:
After trying out some queries, it seems concepts such as `dvc:ProcedureGeboorte` (and other `Procedure*` concepts) that were part of this ttl migrations (`config/migrations/2023/20230228162156-add-new-your-europe-category-codelist/20230228162156-your-europe-categorie.ttl`) were not added correctly to the database.

If I execute the following query, I get 17 top concepts instead of the expected 24.
```
PREFIX dvcs:  <https://productencatalogus.data.vlaanderen.be/id/conceptscheme/>
PREFIX dvc:   <https://productencatalogus.data.vlaanderen.be/id/concept/YourEuropeCategorie/>
PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>

SELECT COUNT(?topConcept) WHERE {
  GRAPH ?g {
    dvcs:YourEuropeCategorie skos:hasTopConcept ?topConcept .
  }
}
```

The deletion migration for the `yourEurope` code-list does contain a large `VALUES` section, but I feel it's the safest option that guarantees everything is correctly deleted.

Reply:
Fine for me; in my head since everything related to this conceptscheme was tied to LPDC; we could do a less verbose query. But anyway we can grind with that i geuss

Reply:
I've trimmed the query down to make it less verbose, and manually deleted the types that were not part of the query.